jenkins/freestyle/delete-old-workers.groovy

```diff
@@ -0,0 +1,19 @@
+import hudson.model.*
+import hudson.node_monitors.*
+import hudson.slaves.*
+import java.util.concurrent.*
+
+import hudson.model.*;
```

Comment:
Actually, are any of the imports needed?
I just commented them all out and ran the script from the jenkins console and it executed successfully without complaint.